drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/utilities/ColumnUtilitiesTest.java

```diff
@@ -146,6 +146,32 @@ public void testGetValueConditionColumn() throws Exception {
         Assertions.assertThat(valueList).containsExactly("1", "-1", "123", "456", "-789");
     }
 
+    @Test
+    public void testIntegerAliases() {
+        column.setFactField(FIELD_NAME);
+        column.setFieldType(DataType.TYPE_NUMERIC_INTEGER);
+        column.setValueList("1=One,2=Two");
+        final String[] valueList = utilities.getValueList(column);
+        Assertions.assertThat(valueList).containsExactly("1=One", "2=Two");
+    }
+
+    @Test
+    public void testBooleanAliases() {
+        column.setFactField(FIELD_NAME);
+        column.setFieldType(DataType.TYPE_BOOLEAN);
+        column.setValueList("true=Yes,false=No");
+        final String[] valueList = utilities.getValueList(column);
+        Assertions.assertThat(valueList).containsExactly("true=Yes", "false=No");
+    }
+
+    @Test
+    public void testBooleanAliasesInvalidValues() {
+        column.setFactField(FIELD_NAME);
+        column.setFieldType(DataType.TYPE_BOOLEAN);
+        column.setValueList("yes=Yes, no=No");
+        assertEquals(0, utilities.getValueList(column).length);
```

Comment:
Can be rewritten as `Assertions.assertThat(utilities.getValueList(column)).isEmpty()`